Jenkinsfile

```diff
@@ -20,35 +20,96 @@ pipeline {
+                                    sh 'mvn clean install -B -q -T 1C -pl !distribution/test/itests/test-itests-ddf'
+                                    sh 'mvn install -B -Dmaven.test.redirectTestOutputToFile=true -pl distribution/test/itests/test-itests-ddf -nsu'
                                 }
                             }
                         }
-                    }, windows: {
-                        node('proxmox-windows'){
+                    },
+                    windows: {
+                        node('proxmox-windows') {
                             timeout(time: 3, unit: 'HOURS') {
                                 checkout scm
                                 withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
-                                    bat 'mvn clean install -pl !distribution/test/itests/test-itests-ddf'
-                                    bat 'mvn install -pl distribution/test/itests/test-itests-ddf -nsu'
+                                    bat 'mvn clean install -B -q -T 1C -pl !distribution/test/itests/test-itests-ddf'
+                                    bat 'mvn install -B -Dmaven.test.redirectTestOutputToFile=true -pl distribution/test/itests/test-itests-ddf -nsu'
                                 }
                             }
                         }
                     }
                 )
             }
         }
+        stage('Static Analysis') {
+            steps {
+                parallel(
+                        owasp: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    sh 'mvn install -q -B -Powasp -DskipTests=true -Dpmd.skip=true -Dfindbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -pl !libs/owasp-diff-runner,!distribution/docs'
+                                }
+                            }
+                        },
+                        sonarqube: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    withCredentials([string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
+                                        sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !platform/solr/platform-solr-server-standalone,!distribution/docs,!distribution/test/itests/test-itests-ddf'
```

Comment:
❓ Is there a way with pipeline scripts to define variables/constants? If so, should we consider extracting the list of modules to skip into one instead of having the list hard-coded here?